src/RotateObject3D.tsx

```diff
@@ -0,0 +1,71 @@
+import * as THREE from 'three';
+import './App.css'
+import { OBJLoader } from 'three/addons/loaders/OBJLoader.js';
+import { MTLLoader } from 'three/addons/loaders/MTLLoader.js';
+
+  
+const RotateObject3D = () => {
```

Comment:
Use `useEffect` from React. 

https://legacy.reactjs.org/docs/hooks-effect.html

Reply:
I added useEffect and now the second object does not appear.